app/controller/command/commands/team.py

```diff
@@ -671,6 +707,42 @@ def refresh_all_team(self):
         else:
             logging.error(f'Could not create {all_name}. Aborting.')
 
+    def refresh_all_rocket_permissions(self):
+        """
+        Refresh Rocket permissions for members in teams like
+        GITHUB_ADMIN_TEAM_NAME and GITHUB_LEADS_TEAM_NAME.
+
+        It only ever promotes users, and does not demote users.
+        """
+        # provide teams from low permissions level to high
+        teams = [
+            {
+                'name': self.config.github_team_leads,
+                'permission': Permissions.team_lead,
+            },
+            {
+                'name': self.config.github_team_admin,
+                'permission': Permissions.admin,
+            },
+        ]
+        for t in teams:
+            if len(t['name']) == 0:
+                continue
+
+            team = None
+            try:
+                team = get_team_by_name(self.facade, t['name'])
+            except LookupError:
+                t_id = str(self.gh.org_create_team(t['name']))
+                logging.info(f'team {t["name"]} created')
+                self.facade.store(Team(t_id, t['name'], t['name']))
+
+            if team is not None:
+                team_members = get_team_members(team)
+                for user in team_members:
+                    user.permissions_level = t['permission']
+                    self.facade.store(user)
```

Comment:
Slight loophole (technically possible, though probably won't be if nobody uses rocket 2):

If the organization has admins appointed already, but do not have an admin team specifically. And if they have a leads team, and they do an update, then all admins who are also team leads will be demoted to team lead status, and because there is no admin team they won't get their permissions back.

Reply:
arghhhhh

Reply:
Wow I even wrote it in the docs!

```py
    def refresh_all_rocket_permissions(self):
        """
        Refresh Rocket permissions for members in teams like
        GITHUB_ADMIN_TEAM_NAME and GITHUB_LEADS_TEAM_NAME.

        It only ever promotes users, and does not demote users.
        """
```

Fix: d0c8b2344858825824b7f510ec33ebef29a1af25